e2e/transactions/transactions_test.go

```diff
@@ -314,6 +314,89 @@ func (s *TransactionsTestSuite) TestSendEtherTx() {
 	s.Zero(s.Backend.TxQueueManager().TransactionQueue().Count(), "tx queue must be empty at this point")
 }
 
+func (s *TransactionsTestSuite) TestSendEtherOnStatusChainTx() {
+
+	s.StartTestBackend(params.StatusChainNetworkID)
+	defer s.StopTestBackend()
+
+	backend := s.LightEthereumService().StatusBackend
+	s.NotNil(backend)
+
+	// create an account
+	sampleAddress, _, _, err := s.Backend.AccountManager().CreateAccount(TestConfig.Account1.Password)
+	s.NoError(err)
+
+	completeQueuedTransaction := make(chan struct{})
+
+	// replace transaction notification handler
+	var txHash = gethcommon.Hash{}
+	signal.SetDefaultNodeNotificationHandler(func(jsonEvent string) { // nolint: dupl
+		var envelope signal.Envelope
+		err := json.Unmarshal([]byte(jsonEvent), &envelope)
+		s.NoError(err, fmt.Sprintf("cannot unmarshal JSON: %s", jsonEvent))
+
+		if envelope.Type == txqueue.EventTransactionQueued {
+			event := envelope.Event.(map[string]interface{})
+			log.Info("transaction queued (will be completed shortly)", "id", event["id"].(string))
+
+			// the first call will fail (we are not logged in, but trying to complete tx)
+			log.Info("trying to complete with no user logged in")
+			txHash, err = s.Backend.CompleteTransaction(
+				common.QueuedTxID(event["id"].(string)),
+				TestConfig.Account1.Password,
+			)
+			s.EqualError(
+				err,
+				account.ErrNoAccountSelected.Error(),
+				fmt.Sprintf("expected error on queued transaction[%v] not thrown", event["id"]),
+			)
+
+			// the second call will also fail (we are logged in as different user)
+			log.Info("trying to complete with invalid user")
+			err = s.Backend.AccountManager().SelectAccount(sampleAddress, TestConfig.Account1.Password)
+			s.NoError(err)
+			txHash, err = s.Backend.CompleteTransaction(
+				common.QueuedTxID(event["id"].(string)), TestConfig.Account1.Password)
+			s.EqualError(
+				err,
+				txqueue.ErrInvalidCompleteTxSender.Error(),
+				fmt.Sprintf("expected error on queued transaction[%v] not thrown", event["id"]),
+			)
+
+			// the third call will work as expected (as we are logged in with correct credentials)
+			log.Info("trying to complete with correct user, this should succeed")
+			s.NoError(s.Backend.AccountManager().SelectAccount(TestConfig.Account1.Address, TestConfig.Account1.Password))
+			txHash, err = s.Backend.CompleteTransaction(
+				common.QueuedTxID(event["id"].(string)),
+				TestConfig.Account1.Password,
+			)
+			s.NoError(err, fmt.Sprintf("cannot complete queued transaction[%v]", event["id"]))
+
+			log.Info("contract transaction complete", "URL", "https://ropsten.etherscan.io/tx/"+txHash.Hex())
```

Comment:
This log is not needed as it's not Ropsten.

Reply:
I agree, will fix it.